main.go

```diff
@@ -28,17 +30,30 @@ func main() {
 		log.Fatal().Err(err).Msg("failed to read github app config")
 	}
 
-	// create client
-	client, err := gh.GetOrganizationClient(ctx, appConfig.Github, config.EnvVars.Org)
-	if err != nil {
-		log.Fatal().Err(err).Msg("failed to register organization client")
-	}
-
 	// create statsd client
 	err = datadog.CreateClient()
 	if err != nil {
 		log.Error().Err(err).Msg("failed to register dogstatsd client")
 	}
 
-	dependabot.Start(ctx, client)
+	// create clients
+	clients, err := gh.GetOrganizationClients(ctx, appConfig.Github)
+	if err != nil {
+		log.Fatal().Err(err).Msg("failed to register organization client")
+	}
+
+	// send stats to dd
+	go datadog.Gauge("organizations", float64(len(clients)), nil)
+
+	// magic will happen
+	for _, client := range clients {
+		wg.Add(1)
+		client := client
+		go func() {
+			dependabot.Start(ctx, client)
+			wg.Done()
```

Comment:
Should we use a `defer` here and then flip line 53 and 54?

Reply:
of course